### ISSUE TITLE
Support Edge

### ISSUE THREAD

**Issue by DevJackSmith** [DevJackSmith/mh-hunt-helper#53]

**Comment by DevJackSmith** [comment]
mostly done, moved to separate repo and submitted request for MS Store

**Comment by DevJackSmith** [comment]
still no response from MS Store

**Comment by tsitu** [comment]
Any update on this and Safari's extension? Do you intend to support these 2 browsers via the CI build pipeline?

**Comment by tehhowch** [comment]
MS Edge extensions are not really worth energy at the moment IMO

**Comment by tsitu** [comment]
FWIW: Edge makes up about 1% of the traffic to my tools, while Safari makes up 13%.

For reference, Chrome has a commanding lead at 73%, while Firefox is third with 8%. If those demos are similar in your tools/extension, then the browser priority list would be Chrome -> Safari -> Firefox, which together represent nearly 95% of users.

**Comment by DevJackSmith** [comment]
with Edge, i had to apply to post on their "addon store". Here's where I applied, no response yet: https://developer.microsoft.com/en-us/microsoft-edge/extensions/requests/

**Comment by DevJackSmith** [comment]
I agree it might not be worth the effort until they figure out what they are doing with Edge. Seems they are replacing it with Chrome base. Once they have it stable and process ironed out, it should be a very easy way to add it to their addon store too.
